### DISCUSSION TITLE
flux cli and customresourcedefinitions

### DISCUSSION THREAD

**Discussion by powdermouse** [fluxcd/flux2#4568] [Q&A]
hi,

we have flux cli updated from v2.1.2 to v2.2.2. after that we get the following error:

**flux_v2.1.2 check**
"► checking prerequisites
✗ flux 2.1.2 <2.2.2 (new version is available, please upgrade)
✔ Kubernetes 1.26.11 >=1.25.0-0
► checking controllers
► checking crds
✔ all checks passed"

**flux_v2.2.2 check**
"► checking prerequisites
✔ Kubernetes 1.26.11 >=1.26.0-0
► checking version in cluster
✗ checking failed: customresourcedefinitions.apiextensions.k8s.io ""gitrepositories.source.toolkit.fluxcd.io"" is forbidden: User ""xxx"" cannot get resource ""customresourcedefinitions"" in API group ""apiextensions.k8s.io"" at the cluster scope
► checking controllers
► checking crds
✗ check failed"

**Comment by stefanprodan** [comment]
That's expected, Flux v2.1.2 had a bug where it would hide query errors. You can't run `flux check`, `flux bootstrap`, `flux install` unless you are a cluster admin and now the CLI tells you that.
